gensim/test/test_word2vec.py

```diff
@@ -461,6 +462,141 @@ def test_cbow_neg(self):
                                   min_count=5, iter=10, workers=2, sample=0)
         self.model_sanity(model)
 
+    def test_most_similar_among_CBOW(self):
```

Comment:
Because `most_similar_among()` functionality is merely on a set of vectors, it's not dependent on the mode-of-training – so there's no need for multiple models built different ways. Testing against one vector set, trained if necessary in the most simple/default way, would be sufficient. (In fact, it'd ultimately make even more sense to be inside a set of KeyedVectors tests, rather than Word2Vec tests, but since its sibling methods are tested here it's not necessary to make that big change yet.)